task5/src/main/java/com/clouway/adapter/persistence/PersistentFundsRepository.java

```diff
@@ -89,4 +94,80 @@ public boolean withdraw(Double amount, String email) {
     }
     return succesfull;
   }
+
+  public void updateHistory(String date, String email, String operation, Double amount) {
+    Connection connection = connectionProvider.get();
+    PreparedStatement statement = null;
+    try {
+      statement = connection.prepareStatement("INSERT INTO transactions (date, email, operation, amount) VALUES (?,?,?,?)");
+      statement.setString(1, date);
+      statement.setString(2, email);
+      statement.setString(3, operation);
+      statement.setDouble(4, amount);
+      statement.execute();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    } finally {
+      try {
+        statement.close();
+      } catch (SQLException e) {
+        e.printStackTrace();
+      }
+    }
+  }
+
+  public void getHistory(PrintWriter out, Integer limit, Integer offset) {
+
+    Connection connection = connectionProvider.get();
+    PreparedStatement statement = null;
+    try {
+      statement = connection.prepareStatement("SELECT * FROM transactions LIMIT " + limit + " OFFSET " + offset + "");
+      ResultSet resultSet = statement.executeQuery();
+      out.print("<table width=25% border=1>");
```

Comment:
In repository method you added `html` code. What if I use `getHistory()` to display in different page?